scripts/dotnet-cli-build/MsiTargets.cs

```diff
@@ -60,6 +69,20 @@ private static void AcquireWix(BuildTargetContext c)
             ZipFile.ExtractToDirectory(Path.Combine(WixRoot, "WixTools.zip"), WixRoot);
         }
 
+        private static void AcquireCrt(BuildTargetContext c)
+        {
+            if (File.Exists(CrtRedistributablePath))
+            {
+                return;
+            }
+
+            Directory.CreateDirectory(Path.GetDirectoryName(CrtRedistributablePath));
+
+            c.Info($"Downloading VC_redist.{Arch}.{CrtVersion}.exe..");
+
+            DownloadFile($"https://dotnetcli.blob.core.windows.net/build/vc/{Arch}/VC_redist.{Arch}.{CrtVersion}.exe", CrtRedistributablePath);
```

Comment:
@piotrpMSFT @gkhanna79 @Petermarcu 

What do you think about the above?  I believe this means you'd need an internet connection in the case where you didn't have the redist installed and went to install the CLI.  It would save us about 10 megs in the bundle, I believe.